os/drivers/lcd/st7789.c

```diff
@@ -799,9 +768,6 @@ static int st7789_init(FAR struct lcd_dev_s *dev)
 
 	st7789_setorientation(dev, ST7789_DEFAULT_ORIENTATION);
 
-	lcd_lock(priv->spi);
```

Comment:
No need when DMA is disabled?

Reply:
yes even when DMA is disabled, it is not needed. It was just to fill initial garbage data with color before any app runs. It can be avoided if app itself turns on lcd or backlight when app is ready to show some valid frames.